public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -136,12 +136,17 @@ mciModule.controller('PerformanceDiscoveryCtrl', function (
       // Load perf data
       .then(function (promise) {
         return dataUtil.getData(
-          promise.fromVersionObj, promise.toVersionObj, expandedCurrent, expandedBaseline, expandedTrend
+          promise.fromVersionObj, promise.toVersionObj, expandedCurrent, expandedBaseline, expandedTrend, metricName
         );
       })
       // Apply perf data
       .then(function (res) {
         vm.gridOptions.data = res
+        let metrics = [];
```

Comment:
Is the following the same?
```
   vm.all_metrics = _.chain(res).map(test => test.all_metrics).uniq().sort().value()
```

Reply:
It's close, I changed it (added a flatten in the middle)

Reply:
:+1: 

I am sometimes dubious about chains but I think it may be clear. (In fact maybe the `.map(test=> test.all_metrics)` could also be replaced with a `.pluck('all_metrics')` but I'm happy with either version.